cfn-resources/project-ip-access-list/cmd/resource/resource.go

```diff
@@ -443,21 +275,21 @@ func isEntryInMap(entry AccessListDefinition, accessListMap map[string]bool) boo
 	return false
 }
 
-func newAccessListMap(accessList []mongodbatlas.ProjectIPAccessList) map[string]bool {
+func newAccessListMap(accessList []admin.NetworkPermissionEntry) map[string]bool {
 	m := make(map[string]bool)
 	for _, entry := range accessList {
-		if entry.CIDRBlock != "" {
-			m[entry.CIDRBlock] = true
+		if *entry.CidrBlock != "" {
```

Comment:
```suggestion
		if util.IsStringPresent(entry.CidrBlock) {
```
To safely check string pointer has a non empty string value. Applies to the following 2 if statements below.

Reply:
updated